compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeWindowDelegate.desktop.kt

```diff
@@ -143,8 +148,40 @@ internal class ComposeWindowDelegate(
                 LocalWindow provides window,
                 LocalLayerContainer provides _pane
             ) {
-                content()
-                undecoratedWindowResizer.Content()
+                val density = LocalDensity.current.density
+                Layout(
+                    {
+                        content()
+                        undecoratedWindowResizer.Content()
+                    },
+                    Modifier,
+                    measurePolicy = { measurables, constraints ->
+                        val contentMeasurable = measurables[0]
+                        val resizerMeasurable = measurables.getOrNull(1)
+
+                        val contentPlaceable = contentMeasurable.measure(constraints)
+                        val width: Int = max(constraints.minWidth, contentPlaceable.width)
+                        val height: Int = max(constraints.minHeight, contentPlaceable.height)
+
+                        val resizerPlaceable = resizerMeasurable?.let{
+                            val resizerWidth = (window.width * density).toInt()
```

Comment:
Better to use `layer.component.density.density` instead of `LocalDensity.current.density`, when we convert values obtained from AWT, as LocalDensity can be overriden by the user (it is not the case here, but it is better to change it for better code)

Reply:
Ok.